scripts/singularity_cmd

```diff
@@ -70,4 +70,12 @@ trap "rm -fr '$tmpdir' && info 'removed temp dir $tmpdir'" exit
 pass_git_config "user.name" "ReproNim User"
 pass_git_config "user.email" "nobody@example.com"
 
-singularity "$cmd" -e -c -W "$tmpdir" -H "$BHOME" -B $PWD --pwd "$PWD" "$@"
+if [ "$OSTYPE" == "linux-gnu" ] && [ -z "${REPRONIM_USE_DOCKER:-}" ]; then
+	singularity "$cmd" -e -c -W "$tmpdir" -H "$BHOME" -B $PWD --pwd "$PWD" "$@"
+else
+	docker run --privileged --rm \
+	-v $PWD:$PWD \
+	-w $PWD \
+	singularityware/singularity:latest \
+	"$cmd" -e -c -B $PWD --pwd $PWD "$@"
```

Comment:
What about -W and -H in this call?

Reply:
@yarikoptic 
Those two switches are not needed. For -W, the system /tmp and /var_tmp are isolated at 2 levels, first by the -c switch we pass and because we are running singularity in a docker container for additional isolation of those system directories from the host. The -H encounters the same isolation as the -W, plus we decided on only $PWD for binding within the singularity container.

Actually, why are we passing those two switches in the non-Docker singularity call? Will the user want to write output to a surrogate home directory?

Reply:
Surrogate home directory provides needed (by git and datalad) .gitconfig and uniform .bashrc . As for /tmp indeed not needed probably, if only for uniform operation

Reply:
In https://github.com/ReproNim/containers/pull/9/files I might also rely on being able to copy bash history for the interactive session from that temporary directory, so would be to have it visible on the host system